### DISCUSSION TITLE
VPN Remote connection

### DISCUSSION THREAD

**Discussion by mendelhart** [ovh/manager#5417] [General]
I would like to connect to my Bare Metal via remote connection.
Is it possible to do?
Can someone please guide me?

Thanks
Mendel H.

**Comment by antleblanc** [comment]
Hi @mendelhart!

Did you already refer to our existing guides?

- https://docs.ovh.com/us/en/dedicated/getting-started-dedicated-server/#logging-on-to-your-server_1
- https://docs.ovh.com/us/en/dedicated/securing-a-dedicated-server/

Hope it will help you,
Regards,
Antoine
